package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@guardian/atom-renderer",
-  "version": "0.17.5",
+  "version": "0.17.6",
```

Comment:
This will be 0.18.1 instead.

Reply:
Yep, that should have changed after the merge master -> resolve conflicts step

Reply:
And it has ✔️